core/src/main/java/com/radixdlt/consensus/bft/BFTSyncer.java

```diff
@@ -88,5 +89,5 @@ enum SyncResult {
    * @param author the original author of the qc
    * @return {@code SyncResult.SYNCED} if already synced
    */
-  SyncResult syncToQC(HighQC highQC, @Nullable NodeId author);
+  SyncResult syncToQC(HighQC highQC, @Nullable NodeId author, HighQcSource highQcSource);
```

Comment:
`if (there are javadoc @params) { update javadoc @params (); }`

Also, it is a good place to indicate that this parameter is not driving any logic, but is only needed for measurement purposes.